packages/file-search/src/browser/quick-file-open.ts

```diff
@@ -30,13 +30,23 @@ import { NavigationLocationService } from '@theia/editor/lib/browser/navigation/
 import * as fuzzy from '@theia/core/shared/fuzzy';
 import { MessageService } from '@theia/core/lib/common/message-service';
 import { FileSystemPreferences } from '@theia/filesystem/lib/browser';
+import { EditorOpenerOptions, Position, Range } from '@theia/editor/lib/browser';
+import { isNumber } from '@theia/core/lib/common/types';
 
 export const quickFileOpen: Command = {
     id: 'file-search.openFile',
     category: 'File',
     label: 'Open File...'
 };
 
+export interface IFilterAndRange {
```

Comment:
Is there a reason for the leading `I`? 
- https://github.com/eclipse-theia/theia/wiki/Coding-Guidelines#interfaces-no-i-prefix

Reply:
Fixed!.  No good reason!, thanks for pointing it out !!